### PR TITLE
Allow reboot from script

### PR DESCRIPTION
**Pull request by olskar**

### CODE REVIEW

File: package/opt/hassbian/suites/upgrade_hassbian.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 function hassbian-show-short-info {
-    echo "Upgrade the base OS installation on this system"
+    echo "Home Assistant upgrade script for Hassbian"
```
Comment (ludeeus) [inline package/opt/hassbian/suites/upgrade_hassbian.sh:3]: It seems that this was reverted to an earlier version?

**Comment by ludeeus** [comment]
This could be an nice feature, but I belive that reboots only should happen when absolutely necessary and they should come organically, most users will hit "y" to every question whiteout reading it, and this will "force" an reboot upon them.

I like the Idea, but if implemented it should only trigger an query when apt reports that reboot is necessary _(If that is even possible?)_

**Comment by Landrash** [comment]
This change introduces a change from current packages and would require user intervention. 
I prefer this change is only applied if the script is run by an actual user.

**Comment by Landrash** [comment]
This functionality will currently not work with the feature set of #90 and forward. This might be included in the future but not in the current iteration. 
Thank you for the PR and if you feel it can be reworked to function with the upcoming changes feel free to drop by for a chat in the #devs-hassbian Discord channel. :cookie:

**Pull request closed without merging**